### DISCUSSION TITLE
Force close (mainly google apps) when opening another app

### DISCUSSION THREAD

**Discussion by dmsfzn** [Mahmud0808/Iconify#608] [Q&A]
Hello, does anybody facing the same issue where after installing iconify, some app such as youtube, playstore, spotify, etc got force close? Even when i disable all overlays the issue still persist. Btw, I'm using magisk alpha latest and lsposed 1.10.0 Jang matrix

**Comment by Mahmud0808** [comment]
Never happened here

**Comment by dmsfzn** [comment]
I forgot, it's kernel issue on my device. Change kernel and that problem is fixed
